spec/rubocop/cop/packaging/relative_require_to_lib_spec.rb

```diff
@@ -59,6 +59,31 @@
     end
   end
 
+  context 'when `require_relative` call is made from inside lib/' do
+    let(:filename) { "#{project_root}/lib/foo.rb" }
+    let(:source) { 'require_relative "../lib/bar"' }
```

Comment:
I think this specific case should be flagged, because it assumes the code is installed to a directory called "lib". However, this is a pretty insane case, and can be fixed later.